### ISSUE TITLE
Weird (probably UTF-8) problem gives false positives on 80char limit.

### ISSUE THREAD

**Issue by vStone** [rodjek/puppet-lint#84]
```
class a {
  # ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
  # ┃          Configuration               ┃
  # ┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
}
```

Although I should probably not use this, it is giving > 80char warnings.
